lib/src/widgets/address_text_field.dart

```diff
@@ -209,8 +211,10 @@ class AddressTextField extends StatelessWidget {
   }
 
   Future<void> _presetAddressBookPicker(BuildContext context) async {
-    final contact = await Navigator.of(context, rootNavigator: true)
-        .pushNamed(Routes.pickerAddressBook,arguments: selectedCurrency);
+    final navigatorState =  ResponsiveLayoutUtil.instance.isMobile(context) ?  Navigator.of(context, rootNavigator: true)
+        :  DesktopDashboardPage.desktopKey.currentState!;
+    final contact = await navigatorState.pushNamed(Routes.pickerAddressBook,arguments: selectedCurrency);
+    
```

Comment:
1- reformat this please
2- why are we using `rootNavigator` in the first place? wouldn't just removing it fix the issue?